rules/game_process.tex

```diff
@@ -415,10 +415,13 @@ \subsubsection{Local Game Stuck}
 \subsubsection{Global Game Stuck}
 \label{sec:game_stuck:global}
 
-If no robots have made progress towards the ball or began to explore the field in \qty{\GameStuckTime}{\second} Global Game Stuck should be called on the team whose robot is \textit{not} nearest the ball.
-The referee calls ``Global Game Stuck \textless color\textgreater''.
+If no robots have made progress towards the ball or began to explore the field in \qty{\GameStuckTime}{\second} Global Game Stuck may be called.
+The referee calls ``Global Game Stuck'' to announce the decision.
+
+Once the referee calls Global Game Stuck, players enter the \texttt{ready} state, and a new kick-off is awarded.
+For the first Global Game Stuck of a half, the kick-off is awarded to the team that did not have the initial kick-off of the half.
+For any further Global Game Stuck calls of a half, kick-off is awarded to the team that did not receive kick-off in to the previous Global Game Stuck call.
```

Comment:
Can't this be shortened: 

The kick-off is awarded to the team that did not have the last (initial) kick-off. 

Also I would add that this is automatically determined by the GC.

Reply:
As disucces on the meeting:

> Also I would add that this is automatically determined by the GC.

Will be added.

> The kick-off is awarded to the team that did not have the last (initial) kick-off.

We won't simplify to Global Game Stuck to keep the kickoff team independent form goals being scored.